client/helpers/get-key-value-pairs.js

```diff
@@ -23,7 +23,7 @@ import moment from 'moment';
 import getJsonStringObject from './get-json-string-object';
 import { jsonKeys, preKeys } from '~constants';
 
-const getKeyValuePairs = event => {
+const getKeyValuePairs = ({ excludes = [], item }) => {
```

Comment:
nit: is the excludes used somewhere other than [] ?

Reply:
It will be used in a future PR.
I can pull from the changeset if needed?

Reply:
also for the js notation:
`({ excludes = [], item }) => {`
This will set `excludes` param to `[]` if not provided a value by default.
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters for more info.

Reply:
that's fine, thank you for providing the more context.